test/schemas/sdkSchemas.test.ts

```diff
@@ -487,6 +488,17 @@ describe('The SDK configuration schema', () => {
             },
             "Expected value of type function at path '/eventProcessor', actual null.",
         ],
+        [
+            {
+                appId: '7e9d59a9-e4b3-45d4-b1c7-48287f1e5e8a',
+                tokenScope: 'global',
+                disableCidMirroring: true,
+                debug: true,
+                test: true,
+                defaultFetchTimeout: 0,
+            },
+            "Expected a value greater than or equal to 1 at path '/defaultFetchTimeout', actual 0.",
```

Comment:
I think it's a matter for a housekeeping PR, but seems like the lint isn't checking for the quote style